README.md

```diff
@@ -335,6 +322,29 @@ tokenName: 'token',
 tokenRoot: false,
 
 
+// Refresh Token Options
+// =====================
+
+// Option to turn refresh tokens On/Off
+useRefreshToken: false,
+// The option to enable/disable the automatic refresh of Auth tokens using Refresh Tokens
+autoUpdateToken: true,
+// The the property from which to get the refresh token after a successful token refresh
+refreshTokenProp: 'refresh_token',
+// Prepended to the `refreshTokenName` when kept in storage (nothing to do with refresh token responses)
+refreshTokenPrefix: 'aurelia',
```

Comment:
Yeah okay so this aligns with [what was said in the other PR](https://github.com/SpoonX/aurelia-authentication/pull/85/files/e8c083f412d22fbbbb3f9023f8b81aacf267a4ad#r58254885). Just checking I'm understanding everything